core/cap-0038.md

```diff
@@ -935,13 +945,21 @@ x = ceil[Xy / (Y - y) / (1 - F)] .
 ```
 In this proposal, `F = 0.003` which corresponds to 0.3%.
 
-#### Upgrades
+#### Ledger Header Flags
 This proposal also adds a `LedgerHeaderExtensionV1` that contains flags for
 validators to vote on using a new `LedgerUpgradeType` `LEDGER_UPGRADE_FLAGS`.
-Only the first flag will be allowed to be set (enforced by
-`MASK_UPGRADE_FLAGS`), which corresponds to `DISABLE_LIQUIDITY_POOL_TRADING`.
-This will allow validators to disable trading against liquidity pools in the
-event that unexpected behavior is encountered.
+Three different flags can be set (enforced by
+`MASK_LEDGERHEADER_FLAGS`), which are -
+
+`DISABLE_LIQUIDITY_POOL_TRADING_FLAG` - disable trading against liquidity pools
```

Comment:
This doesn't render correctly. Write them like 
```
- `DISABLE_LIQUIDITY_POOL_TRADING_FLAG`: disable trading against liquidity pools
```
and it will render as a list.

Reply:
Fixed